tests/tests.rs

```diff
@@ -44,10 +44,12 @@ fn test_infer_bin() {
 
         rls.expect_messages(&[
             ExpectedMessage::new(Some(0)).expect_contains("capabilities"),
-            ExpectedMessage::new(None).expect_contains("beginBuild"),
-            ExpectedMessage::new(None).expect_contains("diagnosticsBegin"),
+            ExpectedMessage::new(None).expect_contains("progress").expect_contains("Build"),
```

Comment:
It's not really important, but could we change it to check against `"title": "Build"` for clarity? (and for others below)